packages/forms/src/elements/FileUpload.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { HTMLAttributes } from 'react';
+import PropTypes from 'prop-types';
+import { StyledFileUpload } from '../styled';
+
+export interface IFileUploadProps extends HTMLAttributes<HTMLDivElement> {
+  /** Apply drag styling programmatically */
```

Comment:
What do you think about "Determines whether to apply drag styling"? Would leaving out the word "programmatically" cause the sentence to lose critical information?

Reply:
Much better. Updated.